docs/source/cluster_management/clis.rst

```diff
@@ -45,4 +45,18 @@ timestamp
 
 Read or recalculate the immutable timestamp. Run ``./bin/atlasdb help timestamp`` for more information.
 
+Offline CLIs
+============
+
+Due to their destructive nature for in-progress transactions, there are two sub-commands of the timestamp CLI which can only be run when AtlasDB is offline. These are:
```

Comment:
In practice the two commands listed _require_ the `--offline` flag, but in order to run a different command while all your clients are down, do you need to specify that flag then in order for them to be able to ignore the config blocks and start embedded stuff?